docs/internal-documentation/maintenance/SystemTesting.md

```diff
@@ -44,58 +44,76 @@ The set of configurations must include at least:
 
 ### Procedure
 
-For each configuration:
-
-1. Prepare a machine
-   1. Allocate a machine with the architecture specified by the configuration.
-   2. Install the OS as specified by the configuration on the machine.
-
-2. Install Zonemaster Engine
-   1. Follow the installation instruction at zonemaster-engine/docs/installation.md
-      to the letter.
-   2. Perform a sanity check.
-
-      ```
-      time perl -MZonemaster -e 'print scalar Zonemaster->test_zone("zonemaster.net"), "\n"'
-      ```
-
-      The command is expected to take very roughly 15 seconds and print a number
-      greater than one.
-
-3. Install Zonemaster Backend
-   1. Follow the installation instruction at zonemaster-backend/docs/installation.md
-      to the letter.
-   2. Perform a sanity check. 
-
-      ```
-      curl -s -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"version_info","id":"1"}' http://localhost:5000/ && echo
-      ```
-
-      The command is expected to give an immediate JSON response.
-
-4. Install Zonemaster GUI
-   1. Follow the installation instruction at zonemaster-gui/docs/installation.md
-      to the letter.
-   2. Perform a sanity check. Point your browser to this URL:
-
-      ```
-      http://localhost/
-      ```
-
-      You should be presented with a page showing, among other things the
-      Zonemaster logotype.
-
-5. Install Zonemaster CLI
-   1. Follow the installation instruction at zonemaster-cli/docs/installation.md
-      to the letter.
-   2. Perform a sanity check.
-
-      ```
-      zonemaster-cli --version
-      ```
-
-      The command is expected to list various version number information.
-
+1. Build a preliminary distribution tarball for each component
+
+2. For each configuration:
+
+   1. Prepare a machine
+      1. Allocate a machine with the architecture specified by the configuration.
+      2. Follow the preparation document for the operating system specified by the configuration.
+         * [FreeBSD-Preparation.md](https://github.com/dotse/zonemaster/blob/master/docs/internal-documentation/distrib-testing/FreeBSD-Preparation.md)
+
```

Comment:
Well, the others haven't been written yet. My idea was that a Debian-preparation document could be extracted from the Debian-System-Testing document of #473. The Ubuntu-preparation document wouldn't differ much from the Debian one - maybe they could even share the same instruction?

@sandoche2k what do you think of this?